lib/Swagger/src/SiteAttribute/Api/AttributeMetadataApi.php

```diff
@@ -0,0 +1,273 @@
+        } catch (ApiException $e) {
+            switch ($e->getCode()) {
+                case 200:
+                    $data = $this->apiClient->getSerializer()->deserialize($e->getResponseBody(), '\Swagger\Client\SiteAttribute\Models\AllAttributesHalResponse', $e->getResponseHeaders());
+                    $e->setResponseObject($data);
+                    break;
+            }
+
+            throw $e;
+        }
+    }
+
+    /**
+     * Operation getSiteAttributeByName
+     *
+     * Returns the value of the given attribute
+     *
+     * @param string $name name of the site attribute (required)
+     * @param int $site_id optional site id (optional)
+     * @return \Swagger\Client\SiteAttribute\Models\AttributeHalResponse
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function getSiteAttributeByName($name, $site_id = null)
+    {
+        list($response) = $this->getSiteAttributeByNameWithHttpInfo($name, $site_id);
+        return $response;
+    }
+
+    /**
+     * Operation getSiteAttributeByNameWithHttpInfo
+     *
+     * Returns the value of the given attribute
+     *
+     * @param string $name name of the site attribute (required)
+     * @param int $site_id optional site id (optional)
+     * @return array of \Swagger\Client\SiteAttribute\Models\AttributeHalResponse, HTTP status code, HTTP response headers (array of strings)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function getSiteAttributeByNameWithHttpInfo($name, $site_id = null)
+    {
+        // verify the required parameter 'name' is set
+        if ($name === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $name when calling getSiteAttributeByName');
+        }
+        // parse inputs
+        $resourcePath = "/attr/{name}";
+        $httpBody = '';
+        $queryParams = array();
+        $headerParams = array();
+        $formParams = array();
+        $_header_accept = $this->apiClient->selectHeaderAccept(array('application/hal+json; charset=UTF-8'));
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(array());
+
+        // query params
+        if ($site_id !== null) {
+            $queryParams['siteId'] = $this->apiClient->getSerializer()->toQueryValue($site_id);
+        }
+        // path params
+        if ($name !== null) {
+            $resourcePath = str_replace(
+                "{" . "name" . "}",
+                $this->apiClient->getSerializer()->toPathValue($name),
+                $resourcePath
+            );
+        }
+        // default format to json
+        $resourcePath = str_replace("{format}", "json", $resourcePath);
+
+        
+        // for model (json/xml)
+        if (isset($_tempBody)) {
+            $httpBody = $_tempBody; // $_tempBody is the method argument, if present
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)